packages/snaps-execution-environments/src/common/BaseSnapExecutor.test.ts

```diff
@@ -666,6 +542,46 @@ describe('BaseSnapExecutor', () => {
     });
   });
 
+  it('only allows certain methods in ethereum API', async () => {
+    const CODE = `
+      module.exports.onRpcRequest = () => ethereum.request({ method: 'snap_blockNumber', params: [] });
```

Comment:
Please use an actually valid snap method for clarity

Reply:
Added changes now.